hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/Message.java

```diff
@@ -81,18 +88,22 @@ public boolean equals(Object obj) {
             return false;
         }
         final Message other = (Message) obj;
-        return Objects.equals(this.offset, other.offset)
-                && Objects.equals(this.partition, other.partition)
-                && Objects.equals(this.topic, other.topic)
+        return Objects.equals(this.topic, other.topic)
```

Comment:
Good point, haven't thought about it, just modified an already existing list of conditions based on all fields. I'll try to change that and see if it breaks anything.